### DISCUSSION TITLE
Pandoc version and library(Flextable)

### DISCUSSION THREAD

**Discussion by Juuso1** [quarto-dev/quarto-cli#3787] [Q&A]
How can I check or update pandoc version in Quarto? My code gives error message:

Error in flextable_to_rmd(x, bookdown = is_bookdown, pandoc2 = pandoc2,  : 
  pandoc version >= 2.0 required for flextable rendering in docx
Calls: .main ... knit_print -> knit_print.flextable -> flextable_to_rmd

BR,
Juuso

**Comment by cscheid** [comment]
`quarto check` should give you the relevant versions:

```
$ quarto check
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 2.19.2: OK
      Dart Sass version 1.32.8: OK
[✓] Checking versions of quarto dependencies......OK
...
```

**Reply by Juuso1** [reply]
Thank you!

Br,
Juuso
